app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -0,0 +1,86 @@
+<form class="ui form {{if isLoading 'loading'}}" autocomplete="off" {{action 'submit' on='submit'}}>
+  {{#if includeSession}}
+    {{#if isSessionSpeaker}}
+      <h3 class="ui header">
+        {{t 'Session detail'}}
+      </h3>
+    {{/if}}
+    {{#each allFields.session as |field|}}
+      {{#if field.isIncluded}}
+        <div class="field">
+          <label class="{{if field.isRequired 'required'}}" for="name">{{field.name}}</label>
+          {{#if (or (eq field.type 'text') (eq field.type 'email'))}}
+            {{#if field.isLongText}}
+              {{widgets/forms/rich-text-editor textareaId=(if field.isRequired (concat 'session_' field.fieldIdentifier '_required'))}}
+            {{else}}
+              {{input type=field.type id=(if field.isRequired (concat 'session_' field.fieldIdentifier '_required'))}}
+            {{/if}}
+          {{/if}}
+          {{#if (eq field.type 'image')}}
+            {{widgets/forms/image-upload
+              label=(t 'Logo')
+              id=(if field.isRequired (concat 'session_' field.fieldIdentifier '_required'))
+              icon='image'
+              hint=(t 'Select an image')
+              maxSizeInKb=1000}}
+          {{/if}}
+          {{#if (eq field.type 'file')}}
+            {{input type=file id=(if field.isRequired (concat 'session_' field.fieldIdentifier '_required'))}}
+          {{/if}}
+          {{#if (eq field.type 'select')}}
+            {{#ui-dropdown class='search selection'}}
+              <i class="dropdown icon"></i>
+              <div class="default text"></div>
+              <div class="menu">
+                <div class="item" data-value="">{{t 'item 1'}}</div>
+              </div>
+            {{/ui-dropdown}}
+          {{/if}}
+        </div>
+      {{/if}}
+    {{/each}}
+  {{/if}}
+
+  {{#if includeSpeaker}}
+    {{#if isSessionSpeaker}}
+      <div class="ui divider"></div>
+      <h3 class="ui header">
+        {{t 'Speaker detail'}}
+      </h3>
+    {{/if}}
+    {{#each allFields.speaker as |field|}}
+      {{#if field.isIncluded}}
+        <div class="field">
+          <label class="{{if field.isRequired 'required'}}" for="name">{{field.name}}</label>
+          {{#if (or (eq field.type 'text') (eq field.type 'email'))}}
+            {{#if field.isLongText}}
+              {{widgets/forms/rich-text-editor textareaId=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required'))}}
+            {{else}}
+              {{input type=field.type id=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required'))}}
```

Comment:
@niranjan94 I will add this component while implementing session creation via API.